requirements.txt

```diff
@@ -1,4 +1,5 @@
-numpy
-scipy
+numpy>=1.7
+scipy>=1.3
 appdirs
-configparser
\ No newline at end of file
+configparser
+packaging
```

Comment:
@edouardoyallon

Reply:
`packaging` allows to use `version` in order to compare versions.

Reply:
Where do we use that?

Reply:
when it does compare the version in the frontends, eg

`kymatio/frontend/torch_frontend.py` line 4.

Reply:
Ah I see. Ok.

What about NumPy and SciPy. Were there issues with earlier versions?

Reply:
I didn't try them but contrary to tf or torch, those frameworks raise warnings like 2 or 3 versions before to even thinking to delete or modify the standard behavior of a feature..(true story, took 3y for one of my code to get obsolete)...

Reply:
So they're very stable? If so, shouldn't we lift the version restrictions on them?

Reply:
So, re-thinking that: what about a simple warning instead of an error, would you be more encline? I'm not really willing to try each possible configurations, but rather to specify a non-outdated, updated and tested code..

Reply:
I think that's good. We should say what versions it's tested for and warn if older versions are used. On the other hand, if we see that it breaks for certain versions, we should put in a minimum version requirement.